ui/pages/confirmations/components/confirm/row/typed-sign-data/typedSignData.test.tsx

```diff
@@ -1,21 +1,29 @@
 import React from 'react';
```

Comment:
Changes in this file look un-intended, may be merge issue.

Reply:
I had to add a mockStore to `typedSignData.test.tsx` and `info.test.tsx` in order for `const isPetNamesEnabled = useSelector(getPetnamesEnabled);` inside `ui/components/app/confirm/info/row/address.tsx` to work correctly.